src/config/chains.ts

```diff
@@ -0,0 +1,89 @@
+// naming in line with subgraph config convention
+const ARBITRUM_ONE = "arbitrum-one"
+const ARBITRUM_SEPOLIA = "arbitrum-sepolia"
+
+class ChainInfo {
+  configName: string;
+  factory: string;
+  wrappedNative: string;
+  wrappedNativeUSDCPool: string;
+  whitelistTokens: Array<string>;
+  stableCoin: string;
+  startBlock: number;
+
+  constructor(
+    configName: string,
+    factory: string,
+    wrappedNative: string,
+    wrappedNativeUSDCPool: string,
+    whitelistTokens: Array<string>,
+    stableCoin: string,
+    startBlock: number
+  ) {
+    this.configName = configName
+    this.factory = factory;
+    this.wrappedNative = wrappedNative;
+    this.wrappedNativeUSDCPool = wrappedNativeUSDCPool;
+    this.whitelistTokens = whitelistTokens;
+    this.stableCoin = stableCoin;
+    this.startBlock = startBlock;
+  }
+}
+
+class SupportedChains {
+  arbitrumOne: ChainInfo;
+  arbitrumSepolia: ChainInfo;
+
+  constructor(arbitrumOne: ChainInfo, arbitrumSepolia: ChainInfo) {
+    this.arbitrumOne = arbitrumOne;
+    this.arbitrumSepolia = arbitrumSepolia;
+  }
+}
+
+const arbitrumOne = new ChainInfo(
```

Comment:
Aside from the Factory, all those addresses (native, WL, etc...) HAVE to be in lowercase. Forcing them into lowercase when applying the conf would avoid potential issues